cli/src/factory_ops/unboxing/init

```diff
@@ -204,13 +204,18 @@ check_mgmt_ip
 # Run pre_unbox validations
 pre_unbox
 
+echo "Running provisioner replace node api" >> ${LOG_FILE}
+provisioner replace_node --node-host 192.168.0.2 --logfile --logfile-filename ${LOG_FILE}
```

Comment:
@83bhp Can we always rely on that hard-coded IP?

Reply:
for now, yes, but can be put in a variable, will do that.